WPCliCS/ruleset.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WPCliCS" xsi:noNamespaceSchemaLocation="../vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+	<description>Coding standard for WP-CLI projects</description>
+
+	<!--
+	To include this ruleset in a WP CLI project, use rule ref="WPCliCS" in brackets.
+
+	For help in using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	For help in understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	-->
+
+	<!--
+	#############################################################################
+	USE THE PHPCompatibility RULESET
+
+	This checks code for PHP cross-version compatibility.
+	See: https://github.com/PHPCompatibility/PHPCompatibility
+	#############################################################################
+	-->
+
+	<rule ref="PHPCompatibility">
+		<!-- Only scan PHP files for PHP compatibility. -->
+		<include-pattern>*\.php$</include-pattern>
+
+		<!-- Polyfill package is included with WP-CLI, so available to all projects. -->
+		<exclude name="PHPCompatibility.FunctionUse.NewFunctions.array_columnFound"/>
+	</rule>
+
+	<!--
+	#############################################################################
+	USE THE WP EXTRA RULESET
```

Comment:
Match name of ruleset similar to the previous one.
```suggestion
	USE THE WORDPRESS EXTRA RULESET
```

Reply:
I've updated it to mirror the actual standard name (in capitalized form).